config/initializers/time_formats.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+Time::DATE_FORMATS[:date_only] = lambda { |time| time.strftime("%A, %B the #{time.day.ordinalize}") }
```

Comment:
I prefer using i18n to set custom date and time formats. See the [i18n docs](https://guides.rubyonrails.org/i18n.html#adding-date-time-formats) for examples.

Reply:
Done... I also renamed it to 'day_and_date' as the more I read it the more I thought 'date_only' was inaccurate